itest/lnd_channel_policy_test.go

```diff
@@ -377,6 +377,12 @@ func testUpdateChannelPolicy(ht *lntest.HarnessTest) {
 		)
 	}
 
+	// Channel Updates actually also have a natural rate limit of 1 update
+	// per second due to the fact that the timestamp carried in the update
+	// is only accurate to the second. So we need to ensure that the next
+	// update we send in the burst is at least 1 second after the last one.
+	time.Sleep(time.Second)
```

Comment:
nit: might make sense to rename assertAliceAndBob to updateAndAssertChanPolicy for clarity too.

Reply:
good idea - will add a commit 🫡